indy/SetStatement300SecondTimeout.btm

```diff
@@ -0,0 +1,25 @@
+# Rules to set a timeout on a Statement reference
+
+RULE Timeout set during Connection.createStatement 
+INTERFACE java.sql.Connection
+METHOD createStatement
+AT RETURN
+IF TRUE
+DO System.out.println("BYTEMAN:set 300s timeout during createStatement");$!.setQueryTimeout(300)
```

Comment:
@whitingjr I'm not quite sure of what this change addresses. Do we have jira or related stuff of the problem we're facing?

Reply:
@ligangty I added you to a ticket that details the context. I hope this provides sufficient information.

Reply:
@whitingjr Sorry and can you send me that ticket again? I'm not sure if I missed that...

Reply:
@ligangty I sent a message on Gchat